venues/OpenReview.net/Support/revisionProcess.py

```diff
@@ -9,7 +9,8 @@ def process(client, note, invitation):
         conference.set_authors()
         conference.set_reviewers()
         if conference.use_area_chairs:
-            conference.set_area_chairs()
+            conference.set_area_chairs(enable_reviewer_reassignment = True)
```

Comment:
we could create a single property set for the switch on/off of the reassignment.